pkg/cloud/gcp/actuators/machine/actuator.go

```diff
@@ -61,12 +62,14 @@ func (a *Actuator) Create(ctx context.Context, machine *machinev1.Machine) error
 		machine:    machine,
 	})
 	if err != nil {
-		return a.handleMachineError(machine, err, createEventAction)
+		fmtErr := fmt.Errorf(scopeFailFmt, machine.GetName(), err)
```

Comment:
wouldn't this shadow the error type breaking consumers?

Reply:
In Go 1.13 it shouldn't, https://blog.golang.org/go1.13-errors